src/Feature/Editor/Draw.re

```diff
@@ -158,22 +158,24 @@ let range =
         -. padding,
       ~height=context.charHeight +. doublePadding,
       ~width=length *. context.charWidth +. doublePadding,
-      ~paint,
+      ~paint=rangePaint,
     );
   };
 };
 
+let textPaint = Skia.Paint.make();
+Skia.Paint.setTextEncoding(textPaint, GlyphId);
+Skia.Paint.setAntiAlias(textPaint, true);
+Skia.Paint.setLcdRenderText(textPaint, true);
```

Comment:
Moving this into a separate `drawShapedText` primitive that can do the shaping as well would be a good idea too I think.

Reply:
Just saw you took care of this for me - thank you @glennsl !